src/main/java/com/google/sps/JobModel.java

```diff
@@ -34,29 +34,37 @@
 // Class used to represent a Job in memory, extracting just the 
 // information required by the analysis
 public abstract class JobModel {
-    public static final int SSD = 1;
-    public static final int HDD = 2;
+    public static final Integer SSD = 1;
+    public static final Integer HDD = 2;
     String projectId;
     String name;
     String id;
+
+    // Type can have following values:
+    // - JOB_TYPE_STREAMING
+    // - JOB_TYPE_UNKNOWN
+    // - JOB_TYPE_BATCH
     String type;
     String sdk = null;
+    // sdkSupportStatus ca be found at 
+    // https://cloud.google.com/dataflow/docs/reference/rest/v1b3/projects.jobs#sdksupportstatus
     String sdkSupportStatus = null;
     String region;
-    String workerLocation;
     int currentWorkers;
```

Comment:
```suggestion
    Integer currentWorkers;
```

Reply:
I think int would more suitable because there is no need for Integer specific methods and also int takes up less space (just 32 bits) in comparison with Integer (128 bits).

Reply:
Ah good point! Never mind then :D